dashboard/src/components/AipReportsCard.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import { ref } from "vue";
+
+import { client, getPath } from "@/client";
+import { ResponseError } from "@/openapi-generator";
+import { useAipStore } from "@/stores/aip";
+import { useAuthStore } from "@/stores/auth";
+
+const aipStore = useAipStore();
+const authStore = useAuthStore();
+const reportKey = ref("");
+const error = ref("");
+
+const downloadDeletionReport = async () => {
+  if (!aipStore.current?.uuid) {
+    return;
+  }
+
+  try {
+    await client.storage.storageAipDeletionReportRequest({
+      uuid: aipStore.current.uuid,
+    });
+    window.open(
+      getPath() + `/storage/aips/${aipStore.current.uuid}/deletion-report`,
+      "_blank",
+    );
+  } catch (err) {
+    // Try to parse the error and save it for 5 seconds. It will
+    // display an alert including the error message.
+    let errorMsg = "Unexpected error downloading deletion report";
+    if (err instanceof ResponseError) {
+      const body = await err.response.json();
+      if (body.message) {
+        errorMsg = body.message;
+      }
+    }
+    error.value = errorMsg;
+    setTimeout(() => (error.value = ""), 5000);
+  }
+};
+
+// Set the deletion report key if the current AIP has been deleted, and the user
+// has permission to download deletion reports.
+if (
+  authStore.checkAttributes(["storage:aips:deletion-report"]) &&
```

Comment:
We should use `storage:aips:deletion:report`, like the existing `storage:aips:deletion:request` and  `storage:aips:deletion:review`, that way you can use the `storage:aips:deletion:*` attribute to allow the three.

Reply:
Sure.